package.json

```diff
@@ -7,29 +7,29 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "configure-mapbox": "scripts/setup-mapbox-sdk-walkthrough.sh",
-    "setupNewDotWebForEmulators": "scripts/setup-newdot-web-emulators.sh",
-    "startAndroidEmulator": "scripts/start-android.sh",
-    "postinstall": "scripts/postInstall.sh",
+    "configure-mapbox": "./scripts/setup-mapbox-sdk-walkthrough.sh",
+    "setupNewDotWebForEmulators": "./scripts/setup-newdot-web-emulators.sh",
+    "startAndroidEmulator": "./scripts/start-android.sh",
+    "postinstall": "./scripts/postInstall.sh",
     "clean": "npx react-native clean-project-auto",
-    "android": "scripts/set-pusher-suffix.sh && npx react-native run-android --mode=developmentDebug --appId=com.expensify.chat.dev --active-arch-only",
-    "ios": "scripts/set-pusher-suffix.sh && npx react-native run-ios --list-devices --mode=\"DebugDevelopment\" --scheme=\"New Expensify Dev\"",
-    "pod-install": "scripts/pod-install.sh",
+    "android": "./scripts/set-pusher-suffix.sh && npx react-native run-android --mode=developmentDebug --appId=com.expensify.chat.dev --active-arch-only",
+    "ios": "./scripts/set-pusher-suffix.sh && npx react-native run-ios --list-devices --mode=\"DebugDevelopment\" --scheme=\"New Expensify Dev\"",
+    "pod-install": "./scripts/pod-install.sh",
     "ipad": "concurrently \"npx react-native run-ios --simulator=\\\"iPad Pro (12.9-inch) (6th generation)\\\" --mode=\\\"DebugDevelopment\\\" --scheme=\\\"New Expensify Dev\\\"\"",
     "ipad-sm": "concurrently \"npx react-native run-ios --simulator=\\\"iPad Pro (11-inch) (4th generation)\\\" --mode=\\\"DebugDevelopment\\\" --scheme=\\\"New Expensify Dev\\\"\"",
     "start": "npx react-native start",
-    "web": "scripts/set-pusher-suffix.sh && concurrently npm:web-proxy npm:web-server",
+    "web": "./scripts/set-pusher-suffix.sh && concurrently npm:web-proxy npm:web-server",
     "web-proxy": "ts-node web/proxy.ts",
     "web-server": "webpack-dev-server --open --config config/webpack/webpack.dev.ts",
     "build": "webpack --config config/webpack/webpack.common.ts --env file=.env.production && ts-node scripts/combine-web-sourcemaps.ts",
```

Comment:
do we need to change these references (L24-L26) to `./scripts` as well?

Reply:
I think so, because while I got this problem in installing the node modules, I asked it in this slack [thread](https://expensify.slack.com/archives/D06DURXSJSW/p1725726591258719) and got instructions to change all usages of `scripts/`  folder files

Reply:
ah okay cool so let's update this PR with the new versions of `build` `build-staging` and `build-adhoc`

Reply:
now shall I resolve this conversation?